docs/pages/guides/tutorials/Hello-Kubernetes-with-Dapr.md

```diff
@@ -0,0 +1,312 @@
+```
+`dapr.io/enabled: true` - this tells the Dapr control plane to inject a sidecar into this deployment.
+
+`dapr.io/app-id: nodeapp` - this assigns a unique id or name to the Dapr application, so it can be sent messages to and communicated with by other Dapr apps.
+
+`dapr.io/enable-api-logging: "true"` - this is added to node.yaml file by default to see the API logs.
+
+1. Import the [nodeapp] file (https://github.com/dapr/quickstarts/blob/master/tutorials/hello-kubernetes/deploy/node.yaml) into the Meshery playground.
+![](./screenshots/dl8.png)
+
+2. `Drag and drop` the nodeapp design to `Hello kubernetes-main`, so that merging designs happen properly.
+![](./screenshots/dl9.png)
+
+3. Click on `Merge` to merge the designs.
+![](./screenshots/dl10.png)
+![](./screenshots/dl11.png)
+
+
+### Step 4: Deploy the Python app using the Dapr sidecar. 
+
+Next, take a quick look at the Python app, which has kubernetes manifest as:
+```\yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pythonapp
+  labels:
+    app: python
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: python
+  template:
+    metadata:
+      labels:
+        app: python
+      annotations:
+        dapr.io/enabled: "true"
+        dapr.io/app-id: "pythonapp"
+        dapr.io/enable-api-logging: "true"
+    spec:
+      containers:
+- name: python
+        image: ghcr.io/dapr/samples/hello-k8s-python:latest
+```
+
+At a quick glance, this is a basic Python app that posts JSON messages to `localhost:3500`, which is the default listening port for Dapr. You can invoke the Node.js application's `neworder` endpoint by posting to `v1.0/invoke/nodeapp/method/neworder`. The message contains some data, including an order ID that increments once per second:
+
+```python
+n = 0
+while True:
+    n += 1
+    message = {"data": {"orderId": n}}
+
+    try:
+        response = requests.post(dapr_url, json=message)
+    except Exception as e:
+        print(e)
+
+    time.sleep(1)
+```
+1. Import the [pythonapp](https://github.com/dapr/quickstarts/blob/master/tutorials/hello-kubernetes/deploy/python.yaml) into the Meshery playground.
+![](./screenshots/dl12.png)
+
+2. `Drag and drop` the Python app design to `Hello kubernetes-main` so that the designs are merged properly.
+![](./screenshots/dl13.png)
+
+3. Click on `Merge` to merge the designs.
+![](./screenshots/dl14.png)
+     
+4. After successfully importing all designs into Meshery, click on `actions` on the left side, then click `deploy` to continue.
+![](./screenshots/dl15.png)
+
+5. Select your environment.
+![](./screenshots/dl16.png)
+
+6. Click on `Deploy` at the final stage so that `Hello kubernetes-main` is successfully deployed.
+![](./screenshots/dl17.png)
+
+#### Step5: Vizualizing the `Hello kubernetes wit dapr`
+
+To view the resources deployed, we will use the **Visualize** section of the _MeshMap_. A view is created with the necessary filters to show the relevant resources.
```

Comment:
Also, what about using the hyperlink that is returned in the deployment notification found in Notification Center? This is a pre-formatted view filter, crafted for show only their design's components.

Reply:
yep , i can include that

Reply:
@deepak4566 please respond to each comment, indicating whether the feedback has been incorporated or not.